.github/workflows/ci.yaml

```diff
@@ -190,47 +196,57 @@ jobs:
         uses: azure/setup-kubectl@v2.1
 
       - name: "[Production] Tag & Push docker images"
-        if: ${{ (github.event_name == 'workflow_dispatch' && inputs.deploy == 'production') || (github.event_name == 'push' && github.ref_type == 'branch' && github.ref_name == 'master') }}
+        if: ${{ env.DEPLOY_PROD == 'true' }}
         run: |
           docker tag image-processor:${{ github.sha }} harbor.disembark.dev/7tv/image-processor:latest
           docker tag image-processor:${{ github.sha }} harbor.disembark.dev/7tv/image-processor:${{ github.sha }}
           docker push harbor.disembark.dev/7tv/image-processor:latest
           docker push harbor.disembark.dev/7tv/image-processor:${{ github.sha }}
 
-      - name: "[Staging] Tag & Push docker images"
-        if: ${{ (github.event_name == 'workflow_dispatch' && inputs.deploy == 'staging') || (github.event_name == 'push' && github.ref_type == 'branch' && github.ref_name == 'dev') }}
-        run: |
-          docker tag image-processor:${{ github.sha }} harbor.disembark.dev/7tv-stage/image-processor:latest
-          docker tag image-processor:${{ github.sha }} harbor.disembark.dev/7tv-stage/image-processor:${{ github.sha }}
-          docker push harbor.disembark.dev/7tv-stage/image-processor:latest
-          docker push harbor.disembark.dev/7tv-stage/image-processor:${{ github.sha }}
+      - name: "[Production] Update deployment template"
```

Comment:
Why do this instead of applying the template then patching the image like it was previously?

Reply:
is because I only want to apply it once
when you do apply, and then patch
you are applying twice.

Reply:
Okay that's fine